var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -229,6 +230,7 @@ class Mfem(Package, CudaPackage, ROCmPackage):
         multi=False,
         description="C++ language standard",
     )
+    variant("memalloc", default=True, description="enable group/batch allocation for some small objects")
```

Comment:
I don't think we need this variant. Please remove it unless you actually know someone who wants to use it.